addons/l10n_it_edi/data/invoice_it_template.xml

```diff
@@ -6,11 +6,11 @@
                 <DettaglioLinee>
                     <NumeroLinea t-esc="line_dict['line_number']"/>
                     <CodiceArticolo t-if="line.product_id.barcode">
-                        <CodiceTipo>EAN</CodiceTipo>
+                        <CodiceTipo t-translation="off">EAN</CodiceTipo>
                         <CodiceValore t-esc="format_alphanumeric(line.product_id.barcode)[:35]"/>
                     </CodiceArticolo>
                     <CodiceArticolo t-elif="line.product_id.default_code">
-                        <CodiceTipo>INTERNAL</CodiceTipo>
+                        <CodiceTipo t-translation="off">INTERNAL</CodiceTipo>
```

Comment:
```suggestion
                        <CodiceTipo t-translation="off">INTERNO</CodiceTipo>
```

Reply:
No, you're right, I don't think he knows it's just a formal code to indicate the meaning of the field. Can you ask hkm?

Reply:
I don't understand. Asking what to who ? It's your task, you should be doing the asking 😃 (I don't have as much understanding/knowledge about it as you and it is more efficient if you ask him/her directly.)

Reply:
Yeah, but I'm still away today, and can't ping him from the phone app. Just asked as a favour to speed up things but ok nvm. :D Will do tomorrow.

Reply:
Just to be sure, is this resolved?

Reply:
What do you mean by "resolved"?
- Resolved as in "retrieved by the translation framework" ? Yes, see the translation I removed here: https://github.com/odoo/odoo/pull/153111/files#diff-8f09903121e8b975eada67a02a995dd1e2cd5fa68f2755147dc7804d1c787e4bL368
- Resolved meaning "solved" - yes, here TSB approves: https://github.com/odoo/odoo/pull/153111#issuecomment-1967266813
- Other meaning that I'm missing?

Reply:
Yes.